app/components/post_list/post_list.js

```diff
@@ -299,30 +299,33 @@ export default class PostList extends PureComponent {
 
     scrollToBottom = () => {
         setTimeout(() => {
-            if (this.flatListRef && this.flatListRef.current) {
+            if (this.flatListRef?.current) {
```

Comment:
`this.flatListRef` shouldn't be null since it's set in the constructor and never cleared. If we're switching to the `?.` operator though, this if block could be
```javascript
this.flatListRef.current?.scrollToOffset({offset: 0, animated: true});
```

Reply:
yes, that is why I always do it this way, this is fine to me

Reply:
I don't think `this.flatListRef` is ever cleared in that case though. React just clears `this.flatListRef.current` in that case as far as I know